packages/cli/src/commands/create/monorepo.ts

```diff
@@ -84,12 +84,37 @@ export const createMonorepo: CreateMonorepoFunc = async ({
     copyFileSync(file, destFilePath)
   })
 
-  console.log(bold(cyan('[info] Running `yarn install`...')))
-  await runCommand(`yarn install`, { cwd: repoDir })
+  if (packageManager === 'yarn' || packageManager === 'pnpm') {
+    const json = JSON.parse(
+      readFileSync(resolve(repoDir, 'package.json')).toString(),
+    ) as Record<string, unknown> & { scripts: Record<string, string> }
+
+    // eslint-disable-next-line functional/immutable-data
+    json['scripts']['add-plugin'] += ` --packageManager ${packageManager}`
+    // eslint-disable-next-line functional/immutable-data
+    json['packageManager'] = packageManager === 'yarn' ? 'yarn@3.2.4' : 'pnpm'
```

Comment:
do we also need to cover npm here?

Reply:
it seems `npm` is the default value if not given.

more info:
Normally people just use npm, yarn or pnpm directly, so this `packageManager` property won't matter much. 
However the recent distributions of Node.js come with something called [corepack](https://github.com/nodejs/corepack) which seems to automatically use specified package manager out of the box.

I didn't know about this until just now! Anyway, so I'd assume npm is the default one, although I haven't found any concrete mention on it.